src/app/_workflows/sitting-record-workflow.service.ts

```diff
@@ -1,5 +1,8 @@
 import { Injectable } from '@angular/core';
 import { FormGroup } from '@angular/forms';
+import { SittingRecordsPostObj, SittingRecordsPostBody } from '../_models/addSittingRecords.model';
```

Comment:
If SittingRecordsPostObj is really not used, remove it?

Reply:
Removed unused import